src/sentry/models/integration.py

```diff
@@ -28,14 +28,14 @@ class DocIntegration(Model):
 
     __include_in_export__ = False
 
-    name = models.CharField(max_length=255)
+    name = models.CharField(max_length=64)
     slug = models.CharField(max_length=64, unique=True)
     author = models.CharField(max_length=255)
     description = models.TextField()
     url = models.URLField()
     popularity = models.PositiveSmallIntegerField(null=True, default=1)
     # Quickly hide/show integration, used for logo uploads
-    is_draft = models.NullBooleanField(default=True)
+    is_draft = models.BooleanField(null=True)
```

Comment:
Is it possible to remove the `null=True` all together? It's pretty much required, and we can retain `default=True` as well